pkg/eventcounter/api/api_test.go

```diff
@@ -59,10 +59,153 @@ func TestNewEventCounterService(t *testing.T) {
 	reg := metrics.DefaultRegisterer()
 	logger, err := log.NewLogger()
 	require.NoError(t, err)
-	g := NewEventCounterService(nil, nil, nil, nil, nil, reg, nil, logger)
+	g := NewEventCounterService(nil, nil, nil, nil, nil, nil, "", reg, nil, logger)
 	assert.IsType(t, &eventCounterService{}, g)
 }
 
+func TestGetEvaluationCountBigquery(t *testing.T) {
+	t.Parallel()
+	mockController := gomock.NewController(t)
+	defer mockController.Finish()
+	now := time.Now()
+
+	patterns := []struct {
+		desc        string
+		setup       func(*eventCounterService)
+		input       *ecproto.GetEvaluationCountV2Request
+		expected    *ecproto.GetEvaluationCountV2Response
+		expectedErr error
+	}{
+		{
+			desc: "error: ErrStartAtRequired",
+			input: &ecproto.GetEvaluationCountV2Request{
+				EnvironmentNamespace: "ns0",
+			},
+			expectedErr: localizedError(statusStartAtRequired, locale.JaJP),
+		},
+		{
+			desc: "error: ErrEndAtRequired",
+			input: &ecproto.GetEvaluationCountV2Request{
+				EnvironmentNamespace: "ns0",
+				StartAt:              now.Add(-7 * 24 * time.Hour).Unix(),
+			},
+			expectedErr: localizedError(statusEndAtRequired, locale.JaJP),
+		},
+		{
+			desc: "error: ErrStartAtIsAfterEndAt",
+			input: &ecproto.GetEvaluationCountV2Request{
+				EnvironmentNamespace: "ns0",
+				StartAt:              now.Unix(),
+				EndAt:                now.Add(-31 * 24 * time.Hour).Unix(),
+			},
+			expectedErr: localizedError(statusStartAtIsAfterEndAt, locale.JaJP),
+		},
+		{
+			desc: "error: ErrFeatureIDRequired",
+			input: &ecproto.GetEvaluationCountV2Request{
+				EnvironmentNamespace: "ns0",
+				StartAt:              now.Add(-30 * 24 * time.Hour).Unix(),
+				EndAt:                now.Unix(),
+			},
+			expectedErr: localizedError(statusFeatureIDRequired, locale.JaJP),
+		},
+		{
+			desc: "success: one variation",
+			setup: func(s *eventCounterService) {
+				s.eventStorage.(*v2ecsmock.MockEventStorage).EXPECT().QueryEvaluationCount(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(
```

Comment:
Please set the parameters using the data from the request, at least for the success cases, instead of `gomock.Any()`.